service/src/test/java/org/databiosphere/workspacedataservice/recordsink/RawlsRecordSinkTest.java

```
@@ -148,6 +148,15 @@ void renamesNameToIncludeRecordType() {
     assertThat(operation.attributeName()).isEqualTo("prefix:widget_name");
   }
 
+  @Test
+  void batchDeleteNotSupported() {
```

New behavior, it'll fail fast if somehow a JSON request with a BatchDelete gets routed to Rawls.  Not that this would have ever happened...